frontend/app/controllers/skill-search.js

```diff
@@ -11,6 +11,13 @@ export default class SkillSearchController extends Controller {
   @service
   router;
 
+  currentSkillId = 0;
+
+  init() {
+    super.init(...arguments);
+    this.set("levelValue", 1);
```

Comment:
this.levelValue = 1
This is kind of confusing because we just switched to Ember Octane. this.set and this.get are still old syntax. Just use the variables like any other instance variable :)

Reply:
This change won't work due to the switch to observers, and the octane observer syntax doesn't work in @classic

Reply:
i see :)